content/news/2020-11-13-release-plans-and-call-for-contrib.md

```diff
@@ -0,0 +1,48 @@
+type = "news"
+title = "R1 Status Update and Call for Contributors"
+author = "bitigchi"
+date = "2020-11-13 13:00:00+03:00"
+tags = ["r1"]
+
+After nearly 20 years of development, Haiku is very close to its first release. We need your help to make it happen. Here's a status update for R1, and latest project news.
+
+## Release Plans
+
+We are planning to release Beta3 sometime next year, hopefully followed by the first stable release, R1. We currently have [31](https://dev.haiku-os.org/query?status=assigned&status=in-progress&status=reopened&status=new&group=status&milestone=R1%2Fbeta3) tickets left for Beta 3, and [688](https://dev.haiku-os.org/query?status=assigned&status=in-progress&status=reopened&status=new&group=status&milestone=R1) tickets for R1.
+
+Haiku project is by no means a small project, it is a full operating system nonetheless. Maintaining a full-stack operating system is a huge work, and needs more than a handful of developers. Recently, one of our core developers, waddlesplash has recently announced his leave from the project due to personal reasons. waddlesplash has contributed to Haiku Project close to a decade, and has been the top contributor for 2017, 2018, and 2019. He has contributed invaluable additions to the Haiku Project, like the NVME driver, and USB3 support, with countless bug fixes and improvements on numerous Haiku components.
```

Comment:
I did not announce it; I commented that I would be significantly scaling things back (i.e. as I already have.) I am still around and paying attention to things, and contributing occasionally. Please do not include me in this post.

Reply:
Thanks, I'll edit accordingly.